blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -77,19 +107,44 @@ function parseArticleItem(item, index, arcSite, phrases, id, customFields) {
         alt_text: altText,
         resized_params: resizedImageOptions = {},
         vanity_credits: vanityCredits,
+        // alignment not always present
+        alignment = '',
       } = item;
 
+      let widthsObject = {
+        small: 768,
+        medium: 1024,
+        large: 1440,
+      };
+
+      // only left and right float supported
+      const allowedFloatValue = alignment === 'left' || alignment === 'right' ? alignment : '';
+
+      if (allowedFloatValue) {
+        // cut the image width in about half if left or right aligned
+        // matched based on allowed widths
+        // the goal was to show 50% of width
+        widthsObject = {
+          small: 274,
+          medium: 400,
+          large: 768,
+        };
+      }
+
       return (url && url.length > 0) ? (
-        <figure key={key}>
+        <FloatableImageContainer
```

Comment:
Curious to the reason for using styled components over classnames here?

Reply:
I can refactor it to use sass; I thought at least the margin and bottom changes could be reusable